internal/bpf/bpfWrapper.c

```diff
@@ -13,10 +13,13 @@
  * limitations under the License.
  */
 
-#include <bpf/xsk.h>	   // for xsk_setup_xdp_prog, bpf_set_link_xdp_fd
 #include <linux/if_link.h> // for XDP_FLAGS_UPDATE_IF_NOEXIST
 #include <net/if.h>	   // for if_nametoindex
-
+#if __has_include(<xdp/xsk.h>)
+#include <xdp/xsk.h>
+#else
+#include <bpf/xsk.h>
+#endif
 #include "bpfWrapper.h"
 #include "log.h"
 

```

Comment:
Additionally in the cleanup prog below
`bpf_set_link_xdp_fd(if_index, fd, XDP_FLAGS_UPDATE_IF_NOEXIST);` will need to be updated to have an option to use libxdp constructs for cleanup using ```xdp_program__detach()``` as shown in: https://github.com/xdp-project/xdp-tools/blob/master/xdp-loader/xdp-loader.c#L248
OR use xdp-loader utility directly to unload the prog:
```
xdp-loader unload veth0 --all
```